api-catalog-services/src/main/resources/application.yml

```diff
@@ -132,6 +132,8 @@ eureka:
                     title: API Catalog
                     description: API Catalog service to display service details and API documentation for discovered API services.
 
+                authentication:
+                    sso: true
```

Comment:
This variable represents support for sso, rather than sso enablement. Should this be named to reflect this purpose better? The goal is to understand the purpose of the config without documentation. This naming is too general to deduce the purpose from the key and value

Reply:
Do `supportsSso` or `ssoSupported` sound better? We can change it in another PR. This has been introduced to enablers in previous PRs.